ci/test/00_setup_env_win64.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) 2019-2020 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+export LC_ALL=C.UTF-8
+
+export CONTAINER_NAME=ci_win64
+export DOCKER_NAME_TAG=ubuntu:18.04  # Check that bionic can cross-compile to win64 (bionic is used in the gitian build as well)
+export HOST=x86_64-w64-mingw32
+export PACKAGES="python3 nsis g++-mingw-w64-x86-64 wine-binfmt wine64"
+export RUN_UNIT_TESTS=true
```

Comment:
Same comment here. Why would we enable unit tests here and not on win32?

Reply:
The tests actually pass in win64, I can still disable it if you want?

Reply:
We should actually try and get the tests to pass on all platforms. I also want to figure out what the qtcharts is giving a missing rule in make. Is there a missing dependency?